WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/BackupProgressChecker.kt

```diff
@@ -1,4 +1,4 @@
-package org.wordpress.android.ui.activitylog
+package org.wordpress.android.ui.jetpack.backup
```

Comment:
Warning (⚠️ ): I am seeing that although the file name was changed to `BackupProgressCheck` the class name is still `BackupDownloadProgressChecker`. Consider removing the `Download` word from the class name as well and have both the file and class name match.

Reply:
@ParaskP7 - Sounds good

Reply:
@ParaskP7  - I talked myself back into using BackupDownload, so all should be consistent again.